wallet-new/src/Cardano/Wallet/Kernel/DB/Spec/Pending.hs

```diff
@@ -203,3 +207,8 @@ liftMap2 f (toMap -> p) (toMap -> p') = fromMap (f p p')
 
 instance Buildable Pending where
     build (Pending (InDb p)) = bprint ("Pending " % mapJson) p
+
+instance Arbitrary Pending where
```

Comment:
I am surprised you can actually write this, this is a bit iffy, as in `module Test.Spec.Submission` I already had to define a `genPending` and that required a proper `ProtocolMagic`:

```haskell
genPending :: Core.ProtocolMagic -> Gen Pending
genPending pMagic = do
    elems <- listOf (do tx  <- Txp.genTx
                        wit <- (V.fromList <$> listOf (Txp.genTxInWitness pMagic))
                        Txp.TxAux <$> pure tx <*> pure wit
                    )
    return $ Pending.fromTransactions elems
```

Maybe you can reuse this generator?